ouroboros-consensus-cardano/README.md

```diff
@@ -3,3 +3,7 @@
 This package contains:
 
 * `src` : Support for Hard Fork between Cardano eras.
+
+* `tools/ledger-db-backends-checker`: simple tool (for the UTxO-HD feature) to
+  compare the contents of an in-memory backing store and an LMDB backing store
+  (i.e., it checks if they have the same slot number and values).
```

Comment:
Remove if https://github.com/input-output-hk/ouroboros-network/pull/4391/files?diff=unified&w=0#r1133748533 is done